src/lib.rs

```diff
@@ -86,8 +86,10 @@ where
 /// which can produce multiple subscribers for provided topics.
 pub fn pubsub_channel<T: Send + Eq, M: Send + Clone>(
     size: usize,
-) -> (TopicPublisher<T, M>, TopicSubscriptionFactory<T, M>) {
-    let (publisher, subscriber): (TopicPublisher<T, M>, TopicSubscriber<T, M>) = bounded(size);
+    receiver_id: usize,
```

Comment:
I think this should actually be a second function that has something like `with_id` in the title. That way when we are done debugging this issue we can just go back to using the original with no extra PR to this repo. In the normal version of this function just set this value to a default 0 and then only log when it is >0